sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/ModelsRepositoryAsyncClient.java

```diff
@@ -62,4 +63,64 @@
     public ModelsRepositoryServiceVersion getServiceVersion() {
         return this.serviceVersion;
     }
+
+    /**
+     * Gets a collection of model definitions.
+     *
+     * @param dtmi A well-formed DTDL model Id. See <a href="https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md">DTDL specs</a>. For example 'dtmi:com:example:Thermostat;1'.
+     * @return A Map containing the model definition(s) and dependencies (if applicable) where the key is the dtmi
+     * and the value is the raw model definition string.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Map<String, String>> getModels(String dtmi) {
```

Comment:
Since this returns a collection of models, should this be a pageable type? Also, generally for listing resources, the prefix used for method name is `list*`.
https://azure.github.io/azure-sdk/java_introduction.html#java-service-client-verbs